auth0/src/test/java/com/auth0/android/authentication/storage/CredentialsManagerTest.kt

```diff
@@ -914,6 +914,34 @@ public class CredentialsManagerTest {
         MatcherAssert.assertThat(manager.hasValidCredentials(), Is.`is`(true))
     }
 
+
+    @Test
+    public fun shouldAddParametersToRequest() {
+        Mockito.`when`(storage.retrieveString("com.auth0.id_token")).thenReturn("idToken")
+        Mockito.`when`(storage.retrieveString("com.auth0.access_token")).thenReturn("accessToken")
+        Mockito.`when`(storage.retrieveString("com.auth0.refresh_token")).thenReturn("refreshToken")
+        val expirationTime = CredentialsMock.ONE_HOUR_AHEAD_MS // non expired credentials
+        val parameters = mapOf(
+            "client_id" to "new Client ID",
+            "phone" to "+1 (777) 124-1588"
+        )
+        Mockito.`when`(storage.retrieveLong("com.auth0.expires_at")).thenReturn(expirationTime)
+        Mockito.`when`(storage.retrieveLong("com.auth0.cache_expires_at"))
+            .thenReturn(expirationTime)
+        Mockito.`when`(storage.retrieveString("com.auth0.scope")).thenReturn("oldscope")
+        Mockito.`when`(
+            client.renewAuth("refreshToken")
+        ).thenReturn(request)
+
+        manager.getCredentials(
+            scope = "some changed scope to trigger refresh",
+            minTtl = 0,
+            parameters = parameters,
+            callback = callback
+        )
+        verify(request).start(requestCallbackCaptor.capture())
```

Comment:
you should as well verify that the request was created with the additional parameters. See for reference what you did in the other test https://github.com/auth0/Auth0.Android/pull/514/files#diff-bd364d0aa7aaac49b5c2c42fc4de0c065f7b9df5c6f8aa8fc50ae665cd057d92R1411